scripts/cli/tsconfig.json

```diff
@@ -1,9 +1,12 @@
 {
   "compilerOptions": {
     "moduleResolution": "node",
-    "module": "commonjs"
+    "module": "es2022",
+    "allowImportingTsExtensions": true,
+    "noEmit": true
   },
   "extends": "../../tsconfig.json",
+  "include": ["./**/*.ts", "./**/*.mts"],
```

Comment:
*mjs?

Reply:
@tomratcliffe  no... im pretty sure i meant the `.mts` - module typescript files. i can't remember if there was something specific that needed this or not.

Reply:
Ah - I was looking at this in isolation, but noticed that `main` also has `reportI18nStats.mjs`